website/addons/forward/tests/factories.py

```diff
@@ -12,6 +12,4 @@ class Meta:
         model = ForwardNodeSettings
 
     owner = SubFactory(ProjectFactory)
-    url = 'http://frozen.pizza.reviews/'
-    redirect_bool = True
-    redirect_secs = 15
+    url = 'http://frozen.pizza.reviews/'
```

Comment:
Trivial, but generally prefer a single newline at EOF. Some editors get upset if it's not there.